hack/quickstart/openstack/config.tf

```diff
@@ -0,0 +1,24 @@
+variable "key_pair" {
+  type    = "string"
+  default = "philips"
```

Comment:
This is just the default value which can be override in terraform using `terraform -var 'key_pair=mykey' apply/plan`, I'll update the readme accordingly.